presto-main/src/main/java/com/facebook/presto/server/RequestErrorTracker.java

```diff
@@ -47,27 +48,36 @@
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 @ThreadSafe
-class RequestErrorTracker
+public class RequestErrorTracker
 {
     private static final Logger log = Logger.get(RequestErrorTracker.class);
 
-    private final TaskId taskId;
-    private final URI taskUri;
+    private final Object id;
```

Comment:
curious if the motivation of moving `taskId` to a more general `id` can be explained a bit in PR?

Reply:
Updated the PR description.